label_studio/ml_models/api.py

```diff
@@ -223,3 +225,51 @@ def get(self, *args):
                 )
         result = {'projects': compatible_project_list}
         return Response(result, status=200)
+
+
+@method_decorator(
+    name='post',
+    decorator=swagger_auto_schema(
+        tags=['Models: Model Runs'],
+        operation_summary='Create Model Run object',
+        operation_description='Create a new Model Run for given project_id, model_version_id and project_subset if it does not exist, if it exists delete the record and create new',
```

Comment:
nit: mind changing the punctuation a bit here? I'm thinking:
Create a new Model Run for given project_id, model_version_id and project_subset if it does not exist. If it exists, delete the record and create new.

Reply:
done